File: mall-website/src/api/member/member.ts
import request, { ApiResult } from '../request'

// 会员实体
export interface Member {
  id: number
  username: string
  nickname?: string
  phone?: string
  gender?: number
  birthday?: string
  avatar?: string
  createTime?: string
}

// 获取会员信息
export const getMemberInfo = (id: number) => {
  return request.get<any, ApiResult<Member>>(`/member/member/${id}`)
}

// 更新会员信息
export const updateMember = (data: Member) => {
  return request.put<any, void>('/member/member', data)
}

File: mall-auth/src/main/java/com/hj/mall/auth/entity/Member.java
package com.hj.mall.auth.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

/**
 * 会员实体（仅包含认证相关字段）
 */
@Data
@TableName("ums_member")
public class Member {

    /**
     * 会员ID
     */
    private Long id;

    /**
     * 用户名
     */
    private String username;

    /**
     * 昵称
     */
    private String nickname;

    /**
     * 手机号
     */
    private String phone;

    /**
     * 邮箱
     */
    private String email;

    /**
     * 头像
     */
    private String avatar;

    /**
     * 账号状态：0-禁用 1-正常
     */
    private Integer status;

    /**
     * 创建时间
     */
    private LocalDateTime createTime;
}
